lib/ui/screens/auth.dart

```diff
@@ -9,213 +9,259 @@ Author: Eric Boysen
-                      child:Padding(
-                        padding: EdgeInsets.fromLTRB(0, 10, 0, 0),
-                        child: RaisedButton(
-                          child:SizedBox(width: 200,
-                          child: Text('continue with Google', textAlign: TextAlign.center,style:GoogleFonts.roboto(textStyle:TextStyle(color:Color(0xFF75b79e), fontWeight: FontWeight.bold,fontSize: 14)))),
-                          onPressed: (){
-                          //for later use with google auth backend
-                        },
-                        color: Color(0xFFEEF9BF),
-                        
-                        )
-                      )
-                    ),
-              ]
+                      Center(
+                          child: Text('OR',
+                              style: GoogleFonts.roboto(
+                                  textStyle:
+                                      TextStyle(color: Color(0xFFEEF9BF))))),
 
-            )
-          )
-        )
-      )
-    );
+                      Align(
+                          //Sign up with Google Authorization
+                          alignment: Alignment.center,
+                          child: Padding(
+                              padding: EdgeInsets.fromLTRB(0, 10, 0, 0),
+                              child: RaisedButton(
+                                child: SizedBox(
+                                    width: 200,
+                                    child: Text('continue with Google',
+                                        textAlign: TextAlign.center,
+                                        style: GoogleFonts.roboto(
+                                            textStyle: TextStyle(
+                                                color: Color(0xFF75b79e),
+                                                fontWeight: FontWeight.bold,
+                                                fontSize: 14)))),
+                                onPressed: () {
+                                  //for later use with google auth backend
+                                },
+                                color: Color(0xFFEEF9BF),
+                              ))),
+                    ])))));
   }
 }
 
-class CustomField extends StatelessWidget{// creates a custom input feild to offer the dark material asthetic instead of general line entry
-  String hintText;//text to be overwritten
-  Function validator;//function to validate data
-  Function onSaved;//function to act upon saving FormState
-  bool isPassword;//if password obscures text
-  bool isEmail;//if email allows email keyboard to appear
-  
-  CustomField({
-    this.hintText,
-    this.validator,
-    this.onSaved,
-    this.isPassword=false,
-    this.isEmail=false
-  });
+class CustomField extends StatelessWidget {
+  // creates a custom input feild to offer the dark material asthetic instead of general line entry
+  String hintText; //text to be overwritten
```

Comment:
Move the comments for these class fields into doc comments.
```suggestion
  /// Information about the field displayed to the user.
  String hintText;
```